File: inflearn/javascript/1-06/main.js
const ps = require('../modules/ps');

const solution = (input) => {
  let answer = '';
  answer = new Set(input);
  return answer.join("");
};

ps.main(async () => {
  let input = await ps.readLine(); // 문자열 입력

  console.log(solution(input));
});

File: inflearn/java/1-06/Main.java
import java.util.*;

public class Main {
    public static String solution(String str) {
        List<Character> answer = new ArrayList<>();
        for (char s : str.toCharArray()) {
            if (!answer.contains(s)) answer.add(s);
        }
        // ArrayList<Character>의 요소를 문자열로 변경 (공백 없음)
        return answer.toString().replaceAll("\\[|, |\\]", "");
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String input = sc.next();

        System.out.println(solution(input));
        sc.close();
    }
}
